Extensions/P2P/B_p2ptools.ts

```diff
@@ -230,10 +238,60 @@ namespace gdjs {
         if (peer === null) return;
         const connection = peer.connect(id);
         connection.on('open', () => {
+          // See line 158 for the failure case.
```

Comment:
Referring to a line is fragile (this will break in the future with the person breaking it not even knowing they broke a comment by adding a new line or doing a refactoring far from this). Can you refer to something else, like a function name or anything else?